migration-engine/migration-engine-tests/src/commands/schema_push.rs

```diff
@@ -11,27 +11,17 @@ pub struct SchemaPush<'a> {
     force: bool,
     /// Purely for logging diagnostics.
     migration_id: Option<&'a str>,
-    rt: Option<&'a tokio::runtime::Runtime>,
+    rt: &'a tokio::runtime::Runtime,
 }
 
 impl<'a> SchemaPush<'a> {
-    pub fn new(api: &'a dyn GenericApi, schema: String) -> Self {
+    pub fn new(api: &'a dyn GenericApi, schema: String, rt: &'a tokio::runtime::Runtime) -> Self {
```

Comment:
sneaky

Reply:
Just deleted `new()` ande renamed `new_sync()` to `new()`.